app/views/templates/script/convert2rhel_analyze.erb

```diff
@@ -25,10 +16,6 @@ if ! rpm -q convert2rhel &> /dev/null; then
   yum install -y convert2rhel
 fi
 
-<% if input('Data telemetry') != "yes" -%>
```

Comment:
Convert2RHEL doesn't collect this env var anymore (removed in https://github.com/oamg/convert2rhel/pull/1102) so having the env var doesn't make any difference.

And about your question: the telemetry is now handled by the [`subscription-manager`](https://github.com/oamg/convert2rhel/blob/main/convert2rhel/subscription.py#L893) and if it fails a warning is printed.

There is also a migration results file saved to `/etc/migration-results`, but I'm not sure how/if it's being collected. Maybe @bocekm would know?

Reply:
As @hosekadam wrote, convert2rhel now always generates a /etc/rhsm/facts/convert2rhel.facts file and then it's up to subscription-manager to upload it to Candlepin (the Red Hat hosted one or the Foreman's on-premise one). So the facts file is collected even if the host registered to Foreman is disconnected from the internet. I'm now talking to a few Satellite folks to make sure that the Satellite's rh_cloud plugin uploads these conversion facts to Insights so that we account for conversions of the systems registered to Satellite in our metrics. If I got it correctly, customers will be able to opt out of this data collection by disabling the rh_cloud plugin.

Reply:
Awesome, thanks